distributionviewer/api/tests/test_views.py

```diff
@@ -14,24 +14,26 @@
 
 class TestMetric(TestCase):
 
-    def create_data(self, date=None):
+    def create_data(self, date=None, population=None):
+        date = date or datetime.date(2016, 1, 1)
+        population = population or 'All'
```

Comment:
Good question. I guess I prefer it since if the default argument is something like a list or a dict that can cause problems. So using `None` is a pattern for those cases.